logic.js

```diff
@@ -0,0 +1,79 @@
+const Tweeter = function () {
+    let _posts = [
```

Comment:
You can use const because you are not changing this variable. Use `const` as much as you can.